packages/emoji-mart-data/i18n/sv.json

```diff
@@ -0,0 +1,29 @@
+{
+  "search": "Sök",
+  "search_no_results_1": "Å nej!",
+  "search_no_results_2": "Den emokin kunde inte hittas",
+  "pick": "Väli en emoji…",
+  "add_custom": "Lägg till anpassad emoji",
+  "categories": {
+    "activity": "Aktivitet",
+    "custom": "Anpassade",
+    "flags": "Flaggor",
+    "foods": "Mat & Dryck",
+    "frequent": "Ofta använd",
+    "nature": "Djur & Natur",
+    "objects": "Föremål",
+    "people": "Smileys och människor",
+    "places": "Resor & platser",
+    "search": "sökresultat",
```

Comment:
```
    "search": "Sökresultat",
```

Reply:
@phedkvist thanks for your review and improvement! they are merged now.

Tack 
Med vänliga hälsningar